CHANGELOG.md

```diff
@@ -63,6 +63,12 @@
   [Martin Redington](https://github.com/mildm8nnered)
   [#5711](https://github.com/realm/SwiftLint/issues/5711)
 
+* Fixes `file_name` rule to match fully-qualified names of nested types.
+  Additionally adds a `require_fully_qualified_names` boolean option to enforce 
+  that file names match nested types only using their fully-qualified name.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.